### PR TITLE
Remove hulk stun punch

### PR DESCRIPTION
**Pull request by optimumtact**
Increase hulk damage

:cl: oranges
tweak: hulk no longer stuns
tweak: John Cena
/:cl:

### CODE REVIEW

**Comment by Dicks1** [comment]
Lame

**Comment by optimumtact** [comment]
Thanks for your feedback

**Comment by Dicks1** [comment]
Stop trying to take away my right to perma stun multiple people at the same time and beat them to death.

**Comment by optimumtact** [comment]
no

**Comment by Dorsisdwarf** [comment]
Drama inbound

**Comment by Jazaen** [comment]
Add the "Drama for the Drama God" label

**Comment by Dorsisdwarf** [comment]
Accurate tagging required cannot tag drama until it starts

**Comment by Cebasiz** [comment]
WOW THIS IS SHIT I LOVE HULK IT IS SO MUCH FUN TO HAVE A BUNCH OF SUPERSTRONG RETARDS RUNNING AROUND RUINING THE STATION 

MAN I DEMAND THIS NOT BE MERGED OR I'LL POST A SHITLOAD OF COMPLAINTS EVERYWHERE

[spoiler] there's the drama [/spoiler]

**Comment by Zilenan91** [comment]
This fixes the Hulk

**Comment by GunHog** [comment]
:-1:

**Comment by Jalleo** [comment]
:unamused: :heavy_multiplication_x:  :8ball:

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm [hunk 1]
```diff
@@ -32,7 +32,7 @@
 /mob/living/carbon/alien/humanoid/attack_hulk(mob/living/carbon/human/user)
 	if(user.a_intent == "harm")
 		..(user, 1)
-		adjustBruteLoss(14 + rand(1,9))
+		adjustBruteLoss(15)
```
Comment (duncathan) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: Wooow unreported nerf right here boyo

**Comment by Durkel** [comment]
Why can't we just replace hulk with something less bullshit? pls.

**Comment by Zilenan91** [comment]
That's what Kor tried to do but he didn't want to deal with the backlash.

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm (hunk 1 above)
Comment (optimumtact) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: it's only a nerf in 8 of 9 cases.

**Comment by optimumtact** [comment]
This isn't controversial, nor is there drama, stop with the meme tags please.

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm [hunk 2]
```diff
@@ -32,7 +32,7 @@
 /mob/living/carbon/alien/humanoid/attack_hulk(mob/living/carbon/human/user)
 	if(user.a_intent == "harm")
 		..(user, 1)
-		adjustBruteLoss(14 + rand(1,9))
+		adjustBruteLoss(15)
 		var/hitverb = "punched"
 		if(mob_size < MOB_SIZE_LARGE)
 			Paralyse(1)
```
Comment (RemieRichards) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: what about the poor xenos...

Comment (optimumtact) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: humans have a small chance to stun on the small alien mobs as well, so I was gonna leave it

Comment (RemieRichards) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: Ah, fair enough then.

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm (hunk 1 above)
Comment (MrPerson) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: wooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooooow

**Comment by Dorsisdwarf** [comment]
There has to actually be drama happening for my memetic tags. Go round up your e-pals and start an argument if you wanna tag it that

**Comment by optimumtact** [comment]
don't encourage lynch mobs

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm (hunk 1 above)
Comment (Fox-McCloud) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: Should be 19.

Comment (optimumtact) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: I'm not changing this line unless a maintainer tells me too

Comment (duncathan) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: Standardize hulk punch at 19 damage :^)

Comment (optimumtact) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: no

**Comment by Shodansbreak** [comment]
How many more nerfs does Hulk need to get?
First they can't use guns
Then they can't use E-swords
Then they yell when they're stunned, revealing that they are hulk.

**Comment by Bluespace** [comment]
eh. 
hulk was a good counter to revs/cult/ops etc, but i'm talking in strictly balance reasons.
hulk could probably be okay to go now.

File: code/modules/mob/living/carbon/alien/humanoid/humanoid.dm (hunk 1 above)
Comment (duncathan) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: >refusing to make a maintainer requested change

DENIED

Comment (optimumtact) [inline code/modules/mob/living/carbon/alien/humanoid/humanoid.dm]: I meant a real maintainer, not a bitch boy who carries their socks and tags pr's

**Comment by duncathan** [comment]
Fuck off oranges I'm a real maintainer :c

**Comment by Zilenan91** [comment]
FREEDOM AT LAST

**Pull request merged**